sky/shell/platform/mac/sky_window.mm

```diff
@@ -100,15 +102,10 @@ - (void)setupAndLoadDart {
 
   auto args = command_line.GetArgs();
   if (args.size() > 0) {
-    _sky_engine->RunFromFile(args[0],
-        command_line.GetSwitchValueASCII(sky::shell::switches::kPackageRoot));
-    return;
-  }
-
-  NSString *endpoint = self.skyInitialBundleURL;
-  if (endpoint.length > 0) {
-    mojo::String string(endpoint.UTF8String);
-    _sky_engine->RunFromBundle(string);
+    _sky_engine->RunFromFile(
+        args[0],  //
```

Comment:
To fool clang-format into not indenting the arguments in a weird way. It usually does an extremely good job. But in this case, it made it look like the last argument to RunFromFile was an argument to GetSwitchValueASCII.